cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -124,7 +127,8 @@
    */
   PRINCIPAL_TYPE("principal-type"),
   PRINCIPAL_NAME("principal-name"),
-  ROLE_NAME("role-name");
+  ROLE_NAME("role-name"),
+  PRINCIPAL("principal");
```

Comment:
the principal-name above is used for authorization to create Principal from the  principal-type which can be user, group or role and the given name.